src/scss/components/_form.scss

```diff
@@ -7,6 +7,34 @@
 **/
 
 .form {
+  position: relative;
+  overflow: hidden;
+
+  &__envelop {
+    background: get-color(reward);
+    width: 100%;
+    position: relative;
+    padding: 30px 30px 20px 30px;
+
+    &__title {
+      margin: 0;
+      text-align: center;
+      color: get-color(air);
+      text-transform: uppercase;
+      font-size: 14px;
+      font-weight: 900;
```

Comment:
@MarcoBrunoBR the variable for font-weight will be created on the fonts-update PR 👍